src/main/java/de/uni_tuebingen/ub/nppm/db/AbstractBase.java

```diff
@@ -493,4 +493,12 @@ public static void verifyDynamicColumn(String column) throws SqlInjectionExcepti
             throw new SqlInjectionException("Invalid column name: " + column);
         }
     }
-}
\ No newline at end of file
+
+    public static String getProvenanceId(String id, String tabelle) throws Exception{
+        return getSingleField("provenance_id", tabelle, Integer.valueOf(id));
+    }
+
+    public static String getProvenanceSrc(String id, String tabelle) throws Exception{
```

Comment:
We don't really use abbreviations in the base yet, so i'd prefer renaming the function to getProvenanceSource.

Reply:
fixed!